tests/unittests/test_handler/test_handler_ca_certs.py

```diff
@@ -201,6 +201,28 @@ def test_single_cert_no_trailing_cr(self):
 
             mock_load.assert_called_once_with("/etc/ca-certificates.conf")
 
+    def test_single_cert_to_empty_existing_ca_file(self):
+        """Test adding a single certificate to the trusted CAs
+        when existing ca-certificates.conf is empty"""
+        cert = "CERT1\nLINE2\nLINE3"
+
+        expected = "cloud-init-ca-certs.crt\n"
+
+        with ExitStack() as mocks:
+            mock_write = mocks.enter_context(
+                mock.patch.object(util, 'write_file'))
```

Comment:
Per [our unit testing guidelines](https://cloudinit.readthedocs.io/en/latest/topics/hacking.html#unit-testing) (search for "An important exception"), if we're going to use the `assert_*` methods on a mock then it needs to be autospecced:

```suggestion
            mock_write = mocks.enter_context(
                mock.patch.object(util, 'write_file', autospec=True))
```